visdiff/src/index.js

```diff
@@ -1,13 +1,19 @@
+#!/usr/bin/env node
+
+import os from 'os'
 import process from 'process'
 import fs from 'fs'
 import path from 'path'
 import crypto from 'crypto'
 import {execSync} from 'child_process'
 import _ from 'lodash'
+import mkdirp from 'mkdirp'
+import del from 'del'
 import gm from 'gm'
 import github from 'octonode'
+import s3 from 's3'
 
-const BUCKET_S3 = 's3://keybase-app-visdiff'
+const BUCKET_S3 = 'keybase-app-visdiff'
 const BUCKET_HTTP = 'http://keybase-app-visdiff.s3.amazonaws.com'
```

Comment:
This is just for formatting the comments on GitHub; the s3 upload goes through the `s3` module which presumably uses SSL. Switching these URLs to be HTTPS wouldn't hurt though, good idea!